bin/fetch_ena.py

```diff
@@ -174,6 +175,8 @@ def parse_args():
     parser.add_argument('--ignore-metadata', action='store_true',
                         help='Download bins instead of MAGs. Does not work if biomes rather than accessions are '
                              'provided in the input file. Default = False')
+    parser.add_argument('--do-not-filter', action='store_true', help='Do not skip genomes that fail QS50 check. '
```

Comment:
maybe `--skip-filter` is more telling name

Reply:
Good idea, I renamed it.